### DISCUSSION TITLE
ESLint couldn't find the config "next/core-web-vitals" to extend from

### DISCUSSION THREAD

**Discussion by Pengy-yay** [super-linter/super-linter#6325] [Q&A] [answered]
When linting file types that use ESLint they all give the same error:
```
 ------
  
  Oops! Something went wrong! :(
  
  ESLint: 8.57.0
  
  ESLint couldn't find the config "next/core-web-vitals" to extend from. Please check that the name of the config is correct.
  
  The config "next/core-web-vitals" was referenced from the config file in "/github/workspace/.eslintrc.json".
  
  If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.
  ------
```
Using default config files & default workflow.
The fixes i've found all fix the issue for self-hosted linting and not super-linter.
I'm new to linting so this is probably an easy fix.
Thanks in advance for the solution!

**Comment by Pengy-yay** [comment, marked as answer]
Solved: ESLint was using the default .eslint.json not inside .github/linter

**Reply by ferrarimarco** [reply]
Thanks for letting us know!
